### PR TITLE
[documentation]: Update Iroha Client CLI readme

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

### CODE REVIEW

File: client_cli/src/main.rs [hunk 1]
```diff
@@ -265,7 +265,7 @@ mod blocks {
 
     use super::*;
 
-    /// Get event stream from iroha peer
+    /// Get block stream from iroha peer
```
Comment (appetrosyan) [inline client_cli/src/main.rs:268]: Excellent catch.

File: client_cli/README.md [hunk 2]
```diff
@@ -119,11 +149,10 @@ cat /path/to/file.wasm | ./iroha_client_cli wasm
 
 These subcommands submit the provided wasm binary as an `Executable` to be executed outside a trigger context.
 
-### Multiple instructions
+### Execute Multiple Instructions
+
+You can execute multiple instructions with a single subcommand. Provide a JSON file with multiple instructions related to Iroha Client CLI to execute:
```
Comment (appetrosyan) [inline client_cli/README.md:154]:
```suggestion
The reference implementation of the Rust client, `iroha_client_cli` is often used for diagnosing problems in other implementations. 

To test transactions in the JSON format (used in the genesis block and by other SDKs), pipe the transaction  into the client and add the `json` subcommand to the arguments: 
```

File: client_cli/README.md [hunk 3]
```diff
@@ -119,11 +149,10 @@ cat /path/to/file.wasm | ./iroha_client_cli wasm
 
 These subcommands submit the provided wasm binary as an `Executable` to be executed outside a trigger context.
 
-### Multiple instructions
+### Execute Multiple Instructions
```
Comment (appetrosyan) [inline client_cli/README.md:152]:
```suggestion
### Execute Multi-instruction Transactions
```

File: client_cli/README.md [hunk 4]
```diff
@@ -20,24 +20,43 @@ cargo build
...
+| `account` | Execute commands related to accounts: register a new one, list all accounts, grant a permission to an account, list all account permissions |
+| `asset`   | Execute commands related to assets: register a new one, mint or transfer assets, get info about an asset, list all assets                   |
+| `blocks`  | Get block stream from Iorha peer                                                                                                            |
+| `domain`  | Execute commands related to domains: register a new one, list all domains                                                                   |
+| `events`  | Get event stream from Iroha peer                                                                                                            |
+| `json`    | Submit multi-instructions as JSON                                                                                                           |
+| `peer`    | Execute commands related to p2p networking: registering and unregistering peers                                                             |
```
Comment (appetrosyan) [inline client_cli/README.md:49]:
```suggestion
| `peer`    | Execute commands related to peer administration and networking                                                             |
```

File: client_cli/README.md [hunk 5]
```diff
@@ -20,24 +20,43 @@ cargo build
...
+| `wasm`    | Execute commands related to WASM                                                                                                            |
+| `help`    | Print the help message for the tool or a subcommand                                                                                         |
```
Comment (appetrosyan) [inline client_cli/README.md:51]:
```suggestion
| `help`    | Print the help message for `iroha_client_cli` and/or the current subcommand other than `help` subcommand                                                                                         |
```

**Review by appetrosyan — DISMISSED** [review]

File: client_cli/README.md [hunk 6]
```diff
@@ -20,24 +20,43 @@ cargo build
...
+| --------------------- | -------------------------------------------------- |
+| -c, --config <config> | Set a config file path (`config.json` by default). |
+
+### Subcommands
+
+|  Command  |                                                                 Description                                                                 |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `account` | Execute commands related to accounts: register a new one, list all accounts, grant a permission to an account, list all account permissions |
+| `asset`   | Execute commands related to assets: register a new one, mint or transfer assets, get info about an asset, list all assets                   |
+| `blocks`  | Get block stream from Iorha peer                                                                                                            |
```
Comment (WRRicht3r) [inline client_cli/README.md:45]:
```suggestion
| `blocks`  | Get block stream from Iroha peer                                                                                                            |
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by 6r1d — DISMISSED** [review]
LGTM

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**